clients/cpp-console/include/DialogManager.h

```diff
@@ -47,6 +47,9 @@ class DialogManager
     void PauseKws();
     void StartKws();
     void StartListening();
+    void StopListening();
+    void MuteListening();
```

Comment:
Per our discussion, have only one API to handle mute & unmute. Maybe call it MuteUnMute()

Reply:
Updated.